fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ColumnRefOperator.java

```diff
@@ -29,6 +34,14 @@ public ColumnRefOperator(int id, Type type, String name, boolean nullable) {
         this.isLambdaArgument = false;
     }
 
+    public void addUsedSubfieldPos(List<Integer> usedNestFieldPos) {
```

Comment:
it's more high cohesion that create a inner class, and put `usedSubfieldPosGroup`/`addUsedSubfieldPos()`/`getUsedSubfieldPosGroup` in it.